sub-pages/_vars-sub-pages.html

```diff
@@ -15,4 +15,9 @@
 {# This to fake data for records sub-pages #}
 {# TODO: Update _queries/records.json when we have real record posts #}
 {% set record_query = queries.records %}
-{% set records = record_query.search_with_url_arguments(size=10) %}
\ No newline at end of file
+{% set records = record_query.search_with_url_arguments(size=10) %}
+
+{# Forms for sub-pages #}
+{% set forms = ['Amending-and-correcting-records-under-the-Privacy-Act-form',
```

Comment:
unfortunately, this is the only way if you have multiple forms on one page. You could just use slugs if there is only one form per page.